src/chocolatey/infrastructure.app/nuget/NugetCommon.cs

```diff
@@ -58,6 +58,7 @@ public static IPackageRepository GetRemoteRepository(ChocolateyConfiguration con
 
             // ensure credentials can be grabbed from configuration
             HttpClient.DefaultCredentialProvider = new ChocolateyNugetCredentialProvider(configuration);
+            HttpClient.DefaultCertificateProvider = new ChocolateyClientCertificateProvider(configuration);

```

Comment:
Same thing here: different layers of authentication, which is actually handy for using existing authorization mechanisms of various servers.